docs/secure-connections/sauce-connect/setup-configuration/ci-cd-integration.md

```diff
@@ -4,6 +4,126 @@ title: Sauce Connect Proxy CI/CD Integration
+
+As an alternative to downloading and installing the SC Client in your CI/CD environment, you can [leverage Docker containers to manage Sauce Connect Proxy tunnels](/secure-connections/sauce-connect/installation/#running-sauce-connect-in-docker). Our Docker image maintained by the Sauce Labs [Open Source Program Office](https://opensource.saucelabs.com/).
+
+Here are some benefits/use cases:
+* If you want to run Sauce Connect Proxy as part of a Dockerized CI/CD.
+* If you'd prefer to manage Docker image tags instead of Sauce Connect Proxy versions.
+* If your setup involves several instances running on the same system, Docker would simplify Sauce Connect Proxy port management.
+
+### Running the SC Docker Image
+
+1. Before running the container, you'll need to pull the Sauce Connect Proxy Docker Image from the Docker Hub. This will always pull the latest version of Sauce Connect Proxy.
+  ```bash
+  $ docker pull saucelabs/sauce-connect
+  ```
+   * Or - if you _do_ want to use specific SC version - you can specify that as a tag:
+   ```bash
+   $ docker pull saucelabs/sauce-connect:4.7.1
+   ```
+2. To run the Sauce Connect Proxy Docker image, execute the below script, which will also set your Sauce Labs username and access key as [environment variables](/basics/environment-variables/).
+  ```bash
+  $ export SAUCE_USERNAME="my-user"
+  $ export SAUCE_ACCESS_KEY="my-access-key"
+  docker run \
+      -e SAUCE_USERNAME=${SAUCE_USERNAME} \
+      -e SAUCE_ACCESS_KEY=${SAUCE_ACCESS_KEY} \
+      -it saucelabs/sauce-connect
+  ```
+
+If desired, you can specify any additional [SC CLI arguments](/dev/cli/sauce-connect-proxy/) here.
+
+::::note
+In Linux, if your tests use localhost addresses, `--network="host"` is required as an argument in the above script to allow requests originating within the container to access the Docker host machine.
+::::
+
+::::warning
+`--network="host"` behaves differently on Windows, Mac OS, and Linux due to the respective platform Docker implementation.
+::::
+
+### Running the Sauce Connect Proxy Docker Image with a CI/CD Pipeline
+If you want to run this Docker image as part of your CI/CD pipeline, you would need a way to determine that Sauce Connect Proxy is ready to proxy the requests. You can achieve that by:
+1. Starting with v4.8.0, the `/readiness` endpoint is available.
+1. By using Docker's [volumes](https://docs.docker.com/storage/volumes/) feature and [--readyfile](/dev/cli/sauce-connect-proxy/#--readyfile).
+
+#### Readiness Endpoint
+::::note
+Only available starting with Sauce Connect Proxy version 4.8.0.
+::::
+
+Starting with 4.8.0, Sauce Connect Proxy Docker image allows configuring liveness and readiness HTTP probes. See the [kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) for more info.
+
+Docker container exposes Sauce Connect Proxy HTTP status server on port 8032. The following endpoints are available:
```

Comment:
```suggestion
When you use one of the following endpoints, the Docker container will expose the Sauce Connect Proxy HTTP status server on port 8032:
```

Reply:
no, docker ALWAYS exposes that port, not just "When you use one of the following endpoints"